generate.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
let's build a CI job that runs this, to ensure we don't edit generated objects

Reply:
Thoughts on switching this to use `go generate`?

Reply:
I personally like the idea of the script, as it lets us quickly build a CI job like I did here: https://github.com/opencost/opencost/pull/2625/files#diff-bd4e577d8cce18da4cecc900fca127f9a0d57d706c9cdb3507b778c1d574a54fR14 that makes sure people don't edit the generated files

Reply:
That's valid, I like that solution.